src/styles/blocks/container.scss

```diff
@@ -0,0 +1,25 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+
+  &__block {
+    display: grid;
+    align-items: center;
+    justify-items: center;
+    height: 300px;
+
+    @each $index, $color in $block-colors {
+      &:nth-child(#{$index}) {
+        background-color: $color;
+      }
+    }
+  }
+
+  @media (min-width: 600px) {
+    grid-template-columns: repeat(2, minmax(300px, 1fr));

```

Comment:
You mentioned that without these two lines the snake only grew at 617px. I have checked with and without these lines and both work the same. Try it again;

Reply:
Wow😮
You're right! I've checked it once again and it works the same both ways. Thank you!